contracts/Exchanger.sol

```diff
@@ -448,6 +480,9 @@ contract Exchanger is Owned, MixinSystemSettings, IExchanger {
         }
     }
 
+    // SIP-115: maybe atomic exchanges shouldn't check the circuit breaker?
+    //          DEX spot can be easily manipulated to suspend a synth, so we would still want to use
+    //          the chainlink rate here
```

Comment:
For this reason, I'm re-considering checking (and potentially tripping) the SIP-65 circuit breaker with atomic exchanges. Perhaps we just check the output amounts against the threshold like SIP-65 settlements (in `_settlementOwing()`) to avoid allowing extremely bad trades?